src/sentry/integrations/issues.py

```diff
@@ -194,6 +195,38 @@ def get_issue_display_name(self, external_issue):
         """
         return ''
 
+    def get_repository_choices(self, group, **kwargs):
+        """
+        Returns the default repository and a set/subset of repositories of asscoaited with the installation
+        """
+        try:
+            repos = self.get_repositories()
+        except ApiError:
+            repo_choices = [(' ', ' ')]
```

Comment:
When the API request fails should we give the user feedback? Right now they'd get an empty list and possibly be confused as to why their repos are gone.